### PR TITLE
Hide mean in violin plot - fixes #15

### LINKED ISSUE #15: Hide mean in violin plot
The violin plot should not show the mean as default. It should at least be optional. In general, the default behavior should be the basic standard plots without unnecessary enhancements.

Example:
```
import uadapy.data as data
import uadapy.dr.uapca as uapca
import uadapy.plotting.plots1D as plots1D

distribs_hi = data.load_iris_normal()
distribs_lo = uapca(distribs_hi, dims=2)
fig, axs = plots1D.plot_1d_distribution(distribs_lo, 50, ['violinplot'], titles=["Dimension 1" ,"Dimension 2"])
```
grafik

### PR DESCRIPTION
**Pull request by Tamir-K**
Change showmeans in violin plot from being always True to a kwarg key
Fixes #15

### CODE REVIEW

**Comment by hageldave** [comment]
Thank you for contributing! I'll quickly review your changes

File: uadapy/plotting/plots1D.py [hunk 1]
```diff
@@ -184,15 +187,16 @@ def plot_1d_distribution(distributions, num_samples, plot_types:list, seed=55, f
...
                         for pc in parts['bodies']:
                             pc.set_facecolor(palette[k % len(palette)])
                             pc.set_edgecolor(palette[k % len(palette)])
                             pc.set_alpha(0.5)
                         parts['cbars'].remove()
                         parts['cmaxes'].remove()
                         parts['cmins'].remove()
-                        parts['cmeans'].set_edgecolor('black')
+                        if kwargs.get('showmeans', False):
+                          parts['cmeans'].set_edgecolor('black')
```
Comment (hageldave) [inline uadapy/plotting/plots1D.py:199]: would it not need to be the other case? if showmeans true -> use black edge color

Comment (Tamir-K) [reply uadapy/plotting/plots1D.py:199]: The "use black edge color" code is only executed if the **showmeans** keyword has been passed into the function as part of kwargs, and is set to true.
The **False** in the second slot of the kwargs.get function is a fallback value to use if the keyword doesn't exist, such as when it hasn't been specified by the user.
I've stuck to the way the **vert** keyword is handled in this function, however, **showmeans** could be made its own parameter, improving usability and readability in both the code and the function's signature.

**Comment by Tamir-K** [comment]
> Thank you for contributing! I'll quickly review your changes

Gladly! Thank you for marking this as a good first issue

**Pull request merged**